tests/vmi_multus_test.go

```diff
@@ -327,11 +333,11 @@ var _ = Describe("Multus", func() {
 				tests.WaitUntilVMIReady(vmiOne, tests.LoggedInAlpineExpecter)
 				tests.WaitUntilVMIReady(vmiTwo, tests.LoggedInAlpineExpecter)
 
-				configInterface(vmiOne, "eth0", "10.1.1.1/24", "localhost:~#")
+				configInterface(vmiOne, "eth0", CIDR1, "localhost:~#")
 				By("checking virtual machine interface eth0 state")
 				checkInterface(vmiOne, "eth0", "localhost:~#")
 
-				configInterface(vmiTwo, "eth0", "10.1.1.2/24", "localhost:~#")
+				configInterface(vmiTwo, "eth0", CIDR2, "localhost:~#")
 				By("checking virtual machine interface eth0 state")
 				checkInterface(vmiTwo, "eth0", "localhost:~#")
 
```

Comment:
you can also use `testIP2` for the `By("ping between virtual machines")` like so:
```
Expect(tests.PingFromVMConsole(vmiOne, testIP2)).To(Succeed())
```

Reply:
Thanks Ram. is that what you have meant?